packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -158,7 +159,7 @@ const CFDsListing = () => {
                             />
                         );
                     })}
-                    {can_get_more_cfd_mt5_accounts && (
+                    {can_get_more_cfd_mt5_accounts && content_flag !== ContentFlag.HIGH_RISK_CR && (
```

Comment:
better to update the `can_get_more_cfd_mt5_accounts` based on this in the traders hub store